File: Java/Day7.java
/*

Day 7: Program to check whether the input received from user is prime or not.

*/

// Program
import java.util.Scanner;

public class Day7{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number to check whether its prime or not: ");
        int n = sc.nextInt();
        
        boolean isPrime = true;

        for(int i = 2; i < n; i++){
            if(n%i == 0){
                isPrime = false;
            }
        }

        if(isPrime){
            System.out.println("Given number is a prime number.");
        }else{
            System.out.println("Given number is not a prime number.");
        }

    
    }
}

/*

Input/Output:

Enter the temperature in Fahrenheit: 
34
Temperature in Celcius is 1.1111111111111112

*/

File: CPP/Day7.cpp
/*

Day6: Program to check if the input recieved from user is prime or not

*/

//Program
#include <iostream>
using namespace std;

int main()
{
    int n;
    cout<<"Enter number: ";
    cin>>n;
    if(n==2 || n==3 || n==5 || n==7)
    {
        cout<<"\nThe number is prime";
    }
    else if(n%2==0 || n%3==0 || n%5==0 || n%7==0 )
    {
        cout<<"\nThe number is not prime";
    }
    else
    {
        cout<<"\nThe number is prime";
    }

    return 0;
}



/*


Input an Output:
Enter number: 199

The number is prime


*/
